examples/md-flexible/scripts/plotPerTraversalAndParameter.py

```diff
@@ -0,0 +1,253 @@
+#!/usr/bin/python3
+
+import os
+import sys
+import plotly.graph_objects as go
+import re
+import numpy
+
+# THIS SCRIPT NEEDS AT LEAST PYTHON 3.8.
+# However, lesser version will probably fail due to invalid syntax instead of this assertion
+assert sys.version_info >= (3, 8)
+
+# ---------------------------------------------- Input ----------------------------------------------
+
+parameterIdentifierString = 'parameter='
+parameterArg = []
+containsParameterArg = False
+
+# help message
+for arg in sys.argv[1:]:
+    if "--help" in arg:
+        print("Usage: ./plotTuning.py parameter=[number, size, density] [path/To//Output/*.out ...]. Meaning number = number of particles, size = boxSize and density = particle density")
+        print("Please use at least python 3.8.")
+        exit(0)
+    elif parameterIdentifierString in arg:
+        parameterArg = arg.split('=', 1)[1]
+        containsParameterArg = True
+    elif not containsParameterArg and not parameterIdentifierString in arg:
+        print("Please specify the parameter for the x-axis: parameter=[number, size, density]")
+        exit(0)
+
+# take all input files as source for a plot
+if len(sys.argv) > 1:
+    if ".out" in arg:
+        datafiles = sys.argv[2:]
+    else:
+        datadirs = sys.argv[2:]
+        datadirs = list(datadirs)
+        datadirs.sort(reverse=True)
+        datafiles = os.listdir(datadirs[0])
+        datafiles = filter(lambda s: s.endswith('.out'), datafiles)
+        datafiles = map(lambda s: datadirs[0] + "/" + s, datafiles)
+        datafiles = list(datafiles)
+
+#    TODO: if nothing is given search for the latest test dir
+
+# ---------------------------------------------- Script ---------------------------------------------
+
+# values for plotting in the end
+allTraversals = []
+allContainers = [[]] * 7
+numberOfParticlesPerTraversal = []
+boxSizePerTraversal = []
+densityPerTraversal = []
+timePerTravsersal = []
+number = False
+size = False
+density = False
+xAxisTitle = "empty"
+xAxis = []
+meanXaxis = []
+meanYaxis = []
+valuesErrorPlus = []
+valuesErrorMinus = []
+
+
+if parameterArg == 'number':
+    number = True
+    xAxisTitle = 'Number of particles'
+elif parameterArg == 'size':
+    size = True
+    xAxisTitle = 'Boxsize'
+elif parameterArg == 'density':
+    density = True
+    xAxisTitle = 'Density of particles'
+
+allContainers[0] = ["directSum"]  # directSum
+allContainers[1] = ["c01", "c08", "c18", "sliced", "c01-combined-SoA", "c04", "c04SoA", "c04HCP"]  # linkedCells
+allContainers[2] = ["verlet-clusters", "verlet-clusters-coloring", "verlet-clusters-static"]  # verletClusterLists
+allContainers[3] = ["verlet-sliced", "verlet-c18", "verlet-c01"]  # verletListsCells
+allContainers[4] = ["verlet-cluster-cells"]  # verletClusterCells
+allContainers[5] = ["var-verlet-lists-as-build"]  # varVerletListsAsBuild
+allContainers[6] = ["verlet-lists"]  # verletLists
```

Comment:
This does not look very sustainable with regard to new containers or traversals. You could generate these arrays from the output of md-flex when you run it once without particles and the `all` arguments.